alpha/lib/model/DeliveryProfileLiveAppleHttp.php

```diff
@@ -147,7 +147,7 @@ protected function isDvrContent($content) {
 		return in_array(self::M3U8_PLAYLIST_END_LIST_IDENTIFIER, array_map('trim', explode("\n", $content)));
 	}
 	
-	public function finalizeUrls(&$baseUrl, &$flavorsUrls)
+	public function disableExtraAttributes(&$baseUrl)
```

Comment:
I would leave DeliveryProfileLiveAppleHttp::finalizeUrls as is instead of changing it to disableExtraAttributes and calling it separately.